manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -176,6 +178,9 @@ spec:
           - name: cacerts
             mountPath: /etc/cacerts
             readOnly: true
+          - name: remote-kube-config
+            mountPath: /etc/remote/config
```

Comment:
please change mountPath to /var/run/secrets/remote to be consistent with the rest of the mountPaths in this file.